packages/angular/projects/ui-angular/src/lib/services/authenticator.service.ts

```diff
@@ -192,6 +192,7 @@ export class AuthenticatorService implements OnDestroy {
   ngOnDestroy(): void {
     if (this._machineSubscription) this._machineSubscription.unsubscribe();
     if (this._unsubscribeHub) this._unsubscribeHub();
+    this._authService.stop();
```

Comment:
nit: should we add a unit test to ensure `stop()` is always being called?

Reply:
Accessing private members seems an existing pattern in the codebase, we can improve this on our side. I'm good with the current test. Thank you for spending time on this!

Reply:
I'm good with this as well. We can clean up the typing on our end